tidb-cloud/tidb-cloud-ip-access-list.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configure IP Access List
+summary: Learn how to configure the root password and allowed IP addresses to connect to your dedicated cluster.
```

Comment:
Has the configuration for the root password been moved elsewhere?

Reply:
yes，Updated